packages/web-components/src/components/cbp-list/cbp-list.scss

```diff
@@ -33,38 +40,29 @@
 }
 
 cbp-list {
-  line-height: var(--cbp-line-height-xs);
-
-  &[size="sm"]:not([variant="link-internal"]):not([variant="link-external"]){
-    font-size: var(--cbp-list-font-small);
+  font-size: var(--cbp-list-font-size);
   
-    li{
-      line-height: var(--cbp-list-item-small-line-height);
-      padding-block: var(--cbp-list-item-small-padding);
+    li{  
+      color: var(--cbp-list-item-color);
+      font-weight: var(--cbp-font-weight-bold);
+      padding-block: var(--cbp-list-item-padding);
+      line-height: var(--cbp-list-item-line-height);
+      padding-block: var(--cbp-list-item-padding-block);
     }
-  }
 
-  &[size="lg"]:not([variant="link-internal"]):not([variant="link-external"]){
-    font-size: var(--cbp-list-font-large);
+  &[size="lg"]:not([variant="link"]){
+    --cbp-list-font-size: var(--cbp-font-size-heading-sm);
   
     li{
```

Comment:
You can set all of these variables at the same level. They don't need to be on the LI selector to inherit down to the LI from the list's selector. This actually complicates overriding the values.